src/query/execute_schema.c

```diff
@@ -10016,6 +10020,366 @@ do_alter_change_tbl_comment (PARSER_CONTEXT * const parser, PT_NODE * const alte
+  MOP class_mop = NULL;
+  OID *usr_oid_array = NULL; //--
+  int user_count = 0;
+  int i;
+  bool has_partitions = false; //--
+  bool is_srv_update_needed = false;
+  OID class_oid;
+  int att_id = -1;
+
+  PT_NODE *attr_node = NULL;
+  PT_NODE *comment_node = NULL;
+  PARSER_VARCHAR *comment = NULL;
+
+  assert (alter_code == PT_CHANGE_COLUMN_COMMENT);
+  assert (alter->info.alter.super.resolution_list == NULL);
+
+  attr_node = alter->info.alter.alter_clause.attr_mthd.attr_def_list;
+  comment_node = attr_node->info.attr_def.comment;
+
+  assert (comment_node != NULL && comment_node->node_type == PT_VALUE);
+
+  comment = comment_node->info.value.data_value.str;
+
+  OID_SET_NULL (&class_oid);
+  reset_att_property_structure (&attr_chg_prop);
+
+  entity_name = alter->info.alter.entity_name->info.name.original;
+  if (entity_name == NULL)
+    {
+      error = ER_UNEXPECTED;
+      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, error, 1, "Expecting a class or virtual class name.");
+      goto exit;
+    }
+
+  class_obj = db_find_class (entity_name);
+  if (class_obj == NULL)
+    {
+      assert (er_errid () != NO_ERROR);
+      error = er_errid ();
+      goto exit;
+    }
+
+  error = locator_flush_class (class_obj);
+  if (error != NO_ERROR)
+    {
+      /* don't overwrite error */
+      goto exit;
+    }
+
+  /* force exclusive lock on class, even though it should have been already acquired */
+  if (locator_fetch_class (class_obj, DB_FETCH_WRITE) == NULL)
+    {
+      error = ER_FAILED;
+      goto exit;
+    }
+
+  ctemplate = dbt_edit_class (class_obj);
+  if (ctemplate == NULL)
+    {
+      /* when dbt_edit_class fails (e.g. because the server unilaterally aborts us), we must record the associated
+       * error message into the parser.	 Otherwise, we may get a confusing error msg of the form: "so_and_so is not a
+       * class". */
+      pt_record_error (parser, parser->statement_number - 1, alter->line_number, alter->column_number, er_msg (), NULL);
+      error = er_errid ();
+      goto exit;
+    }
+
+  error =
+    check_change_attribute (parser, ctemplate, alter->info.alter.alter_clause.attr_mthd.attr_def_list,
+                            alter->info.alter.alter_clause.attr_mthd.attr_old_name,
+                            &(alter->info.alter.constraint_list), &attr_chg_prop, &change_mode);
+  if (error != NO_ERROR)
+    {
+      goto exit;
+    }
+
+  error = tran_system_savepoint (UNIQUE_SAVEPOINT_CHANGE_ATTR);
+  if (error != NO_ERROR)
+    {
+      goto exit;
+    }
+  tran_saved = true;
+
+  error =
+    do_change_att_schema_only (parser, ctemplate, alter->info.alter.alter_clause.attr_mthd.attr_def_list,
+			      alter->info.alter.alter_clause.attr_mthd.attr_old_name,
+			      alter->info.alter.constraint_list, &attr_chg_prop, &change_mode);
+
+  if (error != NO_ERROR)
+    {
+      goto exit;
+    }
+
+  /* save class MOP */
+  class_mop = ctemplate->op;
+
+  /* check foreign key constraints */
```

Comment:
from this line to end of function, the codes loos not needed for checking foreign key constrain and server update need

Reply:
Thank you. But in the last commit, there is no part of /* check foreign key constraints */.